db/castle_db.yml

```diff
@@ -39,161 +46,225 @@ Body:
     Name: Neuschwanstein
     #Name: Noisyubantian
     Npc: Agit#aldeg_cas01
+    Type: First_Edition
+    ClientId: 6
```

Comment:
Personally I would prefer to rewrite all the data in each of the folder re and pre-re to make the db clearer

Reply:
Will do!